meta/templates/base.html

```diff
@@ -44,6 +44,13 @@
     <!-- Estilos -->
     <!-- CSS: implied media="all" -->
     <link rel="stylesheet" href="{{ STATIC_URL }}css/style.css">
+    <link rel="stylesheet" href="{{ STATIC_URL }}css/base.css">
+    <link rel="stylesheet" href="{{ STATIC_URL }}css/nav.css">
+    <link rel="stylesheet" href="{{ STATIC_URL }}css/header.css">
+
+    <link rel="stylesheet" href="{{ STATIC_URL }}css/taxa_page.css">
```

Comment:
Page-specific asset files cannot be in the base, so call assets via the `{% extra-head block %}{% endblock %}` on the specific page